helm/monocular/values.yaml

```diff
@@ -0,0 +1,39 @@
+# Default values for monocular.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+api:
+  replicaCount: 2
+  image:
+    repository: gcr.io/helm-ui/monocular-api
+    tag: latest
+    pullPolicy: Always
+  service:
+    name: monocular-api
+    type: NodePort
+    externalPort: 80
+    internalPort: 8081
```

Comment:
You should be able to run the server with port 80 with the `PORT` env variable.

In any case, I do not think that it is needed.